lib/processor.js

```diff
@@ -42,23 +59,110 @@ const mergeResults = (result) => {
   return {mergeable, description: errorMessages}
 }
 
-const mustInclude = (input, match) => {
-  const regex = new RegExp(match, 'i')
-  const isMergeable = regex.test(input)
+const extractFilterParameter = (params) => {
+  if (typeof params === 'string') return { regex: params }
+
+  const output = {}
+  for (let key in params) {
+    switch (key) {
+      case 'regex':
+        output.regex = params.regex
+        break
+      case 'match':
+        output.match = params.match
+        break
+      case 'message':
+        output.description = params.message
+        break
+      case 'min':
+        output.min = params.min
+        break
+      case 'max':
+        output.max = params.max
+        break
+      case 'enabled':
+        output.enabled = params.enabled
+        break
+      default:
+        break
+    }
+  }
+  return output
+}
+
+const mustInclude = (input, filters) => {
+  let {regex, description} = extractFilterParameter(filters)
+  if (!description) description = `Title does not contain "${regex}"`
+  const isMergeable = new RegExp(regex, 'i').test(input)
+
+  return {
+    mergeable: isMergeable,
+    description: isMergeable ? null : description
+  }
+}
+
+const mustExclude = (input, filters) => {
+  let {regex, description} = extractFilterParameter(filters)
+  if (!description) description = `Title contains "${regex}"`
+  const isMergeable = !(new RegExp(regex, 'i').test(input))
+
+  return {
+    mergeable: isMergeable,
+    description: isMergeable ? null : description
+  }
+}
+const min = (input, filters) => {
+  let {min, description} = extractFilterParameter(filters)
+  if (!description) description = `Assignee count is less than "${min}"`
+  const isMergeable = input.length < min
+
+  return {
+    mergeable: isMergeable,
+    description: isMergeable ? null : description
+  }
+}
+
+const max = (input, filters) => {
+  let {max, description} = extractFilterParameter(filters)
+  if (!description) description = `Assignee count is more than "${max}"`
```

Comment:
I think that the processor should know who the validator ( validator context). In which case the word `Asignee` would be parameterized. Not a big deal since this is currently for title only -- we can refactor when we propagate these across the other validators.

Reply:
I think we can add this when we start working on other validator to use this